Fritz.StreamTools/Views/Followers/Goal.cshtml

```diff
@@ -8,56 +8,57 @@
 
 <html>
 <head>
-		<title>Followers Goal</title>
-		<style>
+	<title>Followers Goal</title>
+	<google-font>@config.FontName</google-font>
+	<style>
 				.followerBar {
 						background: @config.EmptyBackgroundColor;
 						color: @config.EmptyFontColor;
+						font-family: @config.FontName, Arial;
 						}
 
 				#fill {
 						background-image: linear-gradient(to right, @( ViewBag.Gradient));
 						color: @config.FillFontColor;
 				}
-		</style>

```

Comment:
This change looks bigger than it is, because I moved the font-family from the followerbar span up to the section where intellisense doesn't work - so that I don't break intellisense for that whole style section.